stdlib/2/__builtin__.pyi

```diff
@@ -44,6 +45,8 @@ class object:
     def __getattribute__(self, name: str) -> Any: ...
     def __delattr__(self, name: str) -> None: ...
     def __sizeof__(self) -> int: ...
+    def __reduce__(self) -> Union[str, tuple]: ...
```

Comment:
You used Union as a return type. In Mypy, this means that code that uses values returned from this method will trigger errors if the usage is valid for a single type only. This is why in general we're avoiding using Unions as return values.

Is this intended, @vlasovskikh?